frontend/src/pages/Login.tsx

```diff
@@ -7,9 +7,12 @@ import './Login.scss';
 import { IGenericNetworkAction } from 'actions/genericActions';
 import { RootState } from 'reducers/rootReducer';
 import { NEW_PIMS_USER } from 'actionCreators/authActionCreator';
+import useKeycloakWrapper from 'hooks/useKeycloakWrapper';
+import { PROPERTY_VIEW } from 'constants/strings';
 
 const Login = () => {
   const { keycloak } = useKeycloak();
```

Comment:
Could you please share why are we using both keycloak and it's wrapper together here? Wouldn't the wrapper be able to replace all existing functionality?

Reply:
It certainly can.